packages/ui-filecoin/src/SimpleCard.tsx

```diff
@@ -36,38 +38,63 @@ export type SimpleCardData = {
   cta: NonNullable<SimpleCardProps['cta']>
 }
 
-type CardLayoutStyles = Record<
-  BorderKey,
-  { inner: string; content: string; cta: string }
->
-
 type CardContentProps = Pick<SimpleCardProps, 'title' | 'description'> & {
-  layout: CardLayoutStyles[BorderKey]
+  border: CardOuterVariants['border']
 }
 
-type BorderKey = keyof typeof borderStyles
+const cardOuter = cva(['group h-full w-full overflow-hidden'], {
```

Comment:
Does this need to be an array?
```suggestion
const cardOuter = cva('group h-full w-full overflow-hidden', {
```